vaccine/models.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+from django.db import models
+from django.utils import timezone
+from django_prometheus.models import ExportModelOperationsMixin
+from userprofile.validators import za_phone_number
+
+
+class VaccineRegistration(
+    ExportModelOperationsMixin("vaccineregistration"), models.Model
+):
+    GENDER_MALE = "male"
+    GENDER_FEMALE = "female"
+    GENDER_OTHER = "other"
+    GENDER_CHOICES = (
+        (GENDER_MALE, "Male"),
+        (GENDER_FEMALE, "Female"),
+        (GENDER_OTHER, "Other"),
+    )
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    deduplication_id = models.CharField(max_length=255, default=uuid.uuid4, unique=True)
+    msisdn = models.CharField(max_length=255, validators=[za_phone_number])
+    source = models.CharField(max_length=255)
+    gender = models.CharField(
+        max_length=7, choices=GENDER_CHOICES, blank=True, default=""
+    )
+    first_name = models.CharField(max_length=255, blank=True, null=True, default=None)
+    last_name = models.CharField(max_length=255, blank=True, null=True, default=None)
+    date_of_birth = models.DateField(blank=True, null=True, default=None)
+    preferred_time = models.TimeField(blank=True, null=True, default=None)
+    preferred_date = models.DateField(blank=True, null=True, default=None)
+    preferred_location_id = models.CharField(max_length=255)
+    preferred_location_name = models.CharField(max_length=255)
+    id_number = models.CharField(max_length=255, blank=True, default="")
+    refugee_number = models.CharField(
```

Comment:
Could we put an `asylum_seeker_number` on our side? For EVDS we have to put both in the refugee number field, but it might be nice to have them separate on our side.